src/config/constants/farms.ts

```diff
@@ -4385,4 +4385,4 @@ const farms: FarmConfig[] = [
   },
 ]
 
-export default farms
+export default process.env.REACT_APP_USING_FORK === 'true' ? farms.splice(0, 5) : farms
```

Comment:
This is not strictly necessary, but I was facing timeout issues on hardhat due to the high amount of farms/pools